decide-app/src/test/java/com/decide/app/LaunchInterceptorConditionsTest.java

```diff
@@ -160,4 +160,58 @@ public void distanceThrowsIllegalArgumentExceptionOnNull() {
         );
     }
 
+        /**
+     * Positive test case. Ensure the radius of the circle through the
+     * points is correct.
+     */
+    @Test
+    public void findCircleRadiusCorrect() {
+        Point p1 = new Point(1, 0);
+        Point p2 = new Point(2, 1);
+        Point p3 = new Point(3, 0);
+        LaunchInterceptorConditions LIC = new LaunchInterceptorConditions();
+        assertEquals(1, LIC.findCircleRadius(p1, p2, p3), 0.00001);
+    }
+
+    /**
+     * Negative test case. Ensure the radius of the circle through the
+     * points is not incorrect.
+     */
+    @Test
+    public void findCircleRadiusIncorrect() {
+        Point p1 = new Point(1, 0);
+        Point p2 = new Point(2, 1);
+        Point p3 = new Point(5, 0);
+        LaunchInterceptorConditions LIC = new LaunchInterceptorConditions();
+        assertNotEquals(1, LIC.findCircleRadius(p1, p2, p3), 0.00001);
+    }

```

Comment:
Negative test cases on "calculating" methods is a bit redundant, since all it says is "it is not incorrect", in which case it would be better to convert it into a positive test case to say "it is correct".